FirebaseAdmin/FirebaseAdmin.Tests/Auth/FirebaseUserManagerTest.cs

```diff
@@ -86,15 +86,16 @@ public async Task GetUserById()
                     ProjectId = MockProjectId,
                     ClientFactory = factory,
                 });
-            await userManager.GetUserById("user1");
+            UserRecord userRecord = await userManager.GetUserById("user1");
```

Comment:
var userRecord =

Reply:
Yup. I'm so used to Java. Will change that.